src/unix/utilsx11.cpp

```diff
@@ -13,7 +13,9 @@
 // for compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
 
+#if defined(__WXX11__) || (defined(__WXGTK__) && defined(GDK_WINDOWING_X11)) || defined(__WXMOTIF__)
```

Comment:
You will need to move this check below, `GDK_WINDOWING_X11` is definitely not defined here yet.

If you wait until #2261 is merged (or rebase your branch on top of it), you can just do it inside `#if wxHAS_X11_SUPPORT`.